_source/_posts/2023-04-19-api-integrations.md

```diff
@@ -0,0 +1,105 @@
+  2. Your program calls the authorization server's token endpoint to exchange the Client ID and Client Secret for an access token. The access token contains the Okta API Scopes allowed. There may be [Okta SDKs](https://developer.okta.com/code/) that help support this token call for you, depending on your language and framework.
+  > **Note**
+  > If you have a custom authorization server, such as when using a Okta Developer Edition org, you'll have to use signed JSON Web Token (JWT) using a JSON Web Key Set (JWKS). Read more about how to do this on the [Implement OAuth for Okta](https://developer.okta.com/docs/guides/implement-oauth-for-okta-serviceapp/main/) developer documentation.
+  3. When you use the access token to call Okta APIs, you'll add the token to the Authorization header using the `Bearer` scheme.
+
+```http
+GET /api/v1/groups?limit=10 HTTP/1.1
+Authorization: Bearer access_token
+Accept: application/json
+Host: dev-1234567.okta.com
+```
+
+**Considerations with using OAuth**
+
+The preferred method is using OAuth to call Okta APIs. It's more secure, and using widely available standards makes the calling process require less manual code. You'll configure the scopes you want the process to access, which aren't tied to a specific user. But it is a 2-step process to make the API call as first you'll need to get the access token. 
+ 
+
+**When to use OAuth authorization**
+
+You should use OAuth as the preferred authorization mechanism for communication with Okta APIs, including machine-to-machine communications. A widely accepted industry standard like OAuth affords the most security and scope flexibility.
+
+But, if you create an API Service application in Okta to share it outside of your organization, you delegate the scope configuration to the customer as part of their process of setting up an API Service application. There's a temptation for potential customers to either allow too much access or, conversely, limit access, causing frustrations that your integration needs fixes to work as intended. 
+
+**Use API Integration Services**
+
+API Integration Services creates API integration applications using OAuth but with more powers! Your API Integration becomes more discoverable to potential customers with improved handling for scopes. You can submit your integration to the [Okta Integration Network (OIN)](https://developer.okta.com/docs/guides/okta-integration-network/), which makes your integration available in the OIN catalog. Potential customers can find your integration to add to their Okta org. Plus, as part of the submission process, you define the scopes your integration needs beforehand. Customers can see those scopes in the OIN catalog. Transparency in the required resources will help them feel more comfortable using your integration.
```

Comment:
"API Service Integrations" is the label in the Admin Console.
"API service integration" is what we are calling this type of integration in the OIN Manager. This feature only exists if you decide to start the OIN Manager submission process.

Reply:
"Customers can see those scopes in the OIN catalog."  <-- This isn't true today, we don't display scopes in the catalog.
https://www.okta.com/integrations/?capability=api
![Uploading Screen Shot 2023-04-19 at 12.55.38 PM.png…]()

Reply:
@ramyamudduluri-okta when I looked at the OIN catalog and selected an integration to install, it shows me the scopes the integration needs. I can add something about selecting for install before seeing the scope, or are you thinking the phrasing is still incorrect?